ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -1756,20 +1756,21 @@ - (NSNumber *)minimumNumber {
 - (NSNumber *)maximumNumber {
     return @(self.textChoices.count);
 }
-- (NSNumber *)defaultNumber {
+- (id<NSObject, NSCopying, NSCoding>)defaultAnswer {
     if (_defaultIndex < 0 || _defaultIndex >= _textChoices.count) {
         return nil;
     }
-    return @(_defaultIndex);
+   
+    return @[[self.textChoices objectAtIndex:_defaultIndex].value];
 }
 - (NSString *)localizedStringForNumber:(NSNumber *)number {
     return [self.numberFormatter stringFromNumber:number];
 }
 - (NSString *)minimumValueDescription {
-    return ((ORKTextChoice *)[self.textChoices firstObject]).text;
+    return ([self.textChoices firstObject]).text;
```

Comment:
You can use the property format for `firstObject` and `lastObject`, and avoid the parenthesis.